### DISCUSSION TITLE
How to prepare xyz format data(extended xyz file) for training?

### DISCUSSION THREAD

**Discussion by DQM520** [mir-group/nequip#463] [Q&A] [answered]
I use dump command in lammps :1 all custom ${DUMP_FREQ} methane.lammpstrj id type x y z   to get the trajectory. And I convert the trajectory form from .lammpstrj to .xyz using ase, but when I train the model with the ase dataset(.xyz) it goes wrong: **RuntimeError: The field key `total_energy` is not present in this dataset**  , the error says 'total energy is not present in this dataset', but the formal xyz only have particle type and coordinates. Further I knew the extended xyz format is recommanded, but I don't know how to get the formal extended xyz format(Energy、force included). Converted .xyz and error message attached, looking for your valuable reply!
![cbba043649b6a6c0a53010ba704492b](https://github.com/user-attachments/assets/25811df5-0ced-405e-ac6c-09a7df931c77)
![fdefeeff2c49f6e69efd73f1f37c132](https://github.com/user-attachments/assets/49590712-d969-4702-afd5-0fce490097ad)

**Comment by jameskermode** [comment, marked as answer]
Question asked again and answered at https://github.com/libAtoms/QUIP/issues/207

**Reply by Linux-cpp-lisp** [reply]
Thanks for answering and linking @jameskermode , appreciate it!

**Reply by DQM520** [reply]
> Question asked again and answered at [libAtoms/QUIP#207](https://github.com/libAtoms/QUIP/issues/207)

I have solved this problem, thank you very much!
